runtime/heiko/src/lib.rs

```diff
@@ -929,9 +970,101 @@ pub type LocationToAccountId = (
     AccountId32Aliases<RelayNetwork, AccountId>,
 );
 
+// Our AssetType. For now we only handle Xcm Assets
+#[derive(Clone, Eq, Debug, PartialEq, Ord, PartialOrd, Encode, Decode, TypeInfo)]
+pub enum AssetType {
```

Comment:
Wondering if this is really needed for now?

Reply:
yes, actually it is just the representation of `Multilocation`

Reply:
@rocketman1997 I feel we can use MultiLocation at the beginning to make this pallet smaller and easier

Reply:
@GopherJ yes,with `MultiLocation` is more fucused to xcm but one benifit with the abstraction `AssetType` provides better extendibility the [enum](https://github.com/rocketman1997/parallel-1/blob/9508db5cf074b53e4ae3d18618d6f2614a479533/runtime/heiko/src/lib.rs#L975-L977)  could be extended to none xcm/substrate based chains